packages/salesforcedx-utils-vscode/src/services/telemetry.ts

```diff
@@ -383,4 +386,60 @@ export class TelemetryService {
         .catch(err => console.error(err));
     }
   }
+
+  private getRandomUserId = (): string => randomBytes(20).toString('hex');
+
+  private buildCliTelemetryCommand(): Command {
+    return new SfCommandBuilder()
+      .withArg('telemetry')
+      .withJson()
+      .build();
+  }
+
+  private async executeCliTelemetry(
```

Comment:
@RitamAgrawal given that this function sole purpose is to run the telemetry command I would suggest creating the Command inside executeCliTelemetry instead having two functions.

Reply:
@peternhale okay! I see that's how we are implementing execute in other classes as well. I have added tests for buildTelemetryCommand  as well. Do you think it's necessary to combine the two fn?